docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
+RUN \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/ndelvalle/generator-api/pullRequest?prid=6300885)

Reply:
this will be too much, Packages used are common ones